DailyReportWeb/Models/PolicySummary.cs

```diff
@@ -7,8 +7,11 @@ namespace DailyReportWeb.Models
 {
     public class PolicySummary
     {
-        public string Id { get; set; }
-        public string ProjectName { get; set; }
-        public string ReportTime { get; set; } 
+        public string id { get; set; }
+        public string key { get; set; }
+        public string name { get; set; }
+        public string time { get; set; }
+        public string baseUrl { get; set; }
+        public string sharedSecret { get; set; }
```

Comment:
deserializarea nu va mai functiona din json > c# object
putem face set Id = id, moment in care o instanta a clasei policySummary, va avea ps.id si ps.Id